Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/durationpicker/DurationExtension.kt

```diff
@@ -3,18 +3,8 @@ package de.rki.coronawarnapp.ui.durationpicker
 import org.joda.time.Duration
 
 fun Duration.toContactDiaryFormat(): String {
```

Comment:
Maybe add tests to verify if this extension still works as expected?

Reply:
One more test is never a bad idea 😃